docs/server-instructions.md

```diff
@@ -0,0 +1,102 @@
+# Add the JVM agent to Java servers
+
+The following sections show how to add the path to the Splunk OTel agent for Java using the [supported servers](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#application-servers).
+
+## JBoss EAP / WildFly
+
+Add the `javaagent` argument to the `standalone` configuration file:
+
+- On Linux and macOS, add the following line at the end of the `standalone.conf` file:
+   ```
+   JAVA_OPTS="$JAVA_OPTS -javaagent:/path/to/splunk-otel-javaagent.jar"
+   ```
+- On Windows, dd the following line at the end of the `standalone.conf.bat` file:
+   ```
+   set "JAVA_OPTS=%JAVA_OPTS% -javaagent:<Drive>:\path\to\splunk-otel-javaagent.jar"
+   ```
+
+## Jetty
+
+Configure the path to the JVM agent either in `start.ini` or in `jetty.sh`:
```

Comment:
Jetty documentation https://www.eclipse.org/jetty/documentation/jetty-9/index.html#quickstart-running-jetty uses `java -jar start.jar`. As does documentation for jetty 10 https://www.eclipse.org/jetty/documentation/jetty-10/operations-guide/index.html#og-start I think mentioning `jetty.sh` is ok. Additionally should also mention what to do when using `java -jar start.jar`. `start.ini` part should be deemphasized (or removed) using language that suggest to use it only when user already has other jvm arguments there.

Reply:
What would the command look like? `java -javaagent:/path/to/splunk-otel-javaagent.jar -jar <jetty_home>/start.jar` ?

Reply:
I would drop `<jetty_home>/` jetty doc also has just `java -jar start.jar`